bftengine/tests/SigManager/SigManager_test.cpp

```diff
@@ -168,7 +171,9 @@ TEST(SigManagerTest, ReplicasOnlyCheckVerify) {
     ASSERT_FALSE(sigManager->verifySig(i, data1, RANDOM_DATA_SIZE, sig.data(), expectedVerifierSigLen));
 
     // change signature randomally, expect failure
-    string sig1(sig);
+    std::string sig1;
```

Comment:
The ctor does `reserve` automatically. What is the point to do it manually here?

Reply:
Modified the code. Committed here https://github.com/vmware/concord-bft/pull/1750/commits/4ad251175ecbc63aca1f7fcfee3e90d01fa170f5